Irrgarten/src/irrgarten/Game.java

```diff
@@ -0,0 +1,112 @@
+package irrgarten;
+import java.util.ArrayList;
+
+public class Game {
+    public static int MAX_ROUNDS = 10;
+    private int currentPlayerIndex;
+    private String log;
+
+    private Player currentPlayer;
+    private Labyrinth labyrinth;
+    private ArrayList<Monster> monsters;
+    private ArrayList<Player> players;
+
+    // Parametrized
+    public Game(ArrayList<Player> players, ArrayList<Monster> monsters) {
+	this.players = players;
+	this.monsters = monsters;
+	// Escojo aleatoriamente un jugador para empezar
+	this.currentPlayerIndex = Dice.whoStarts(players.size());
+	this.currentPlayer = this.players.get(this.currentPlayerIndex);
+	
+	this.log = "";
+	this.labyrinth = new Labyrinth();
+	this.configureLabyrinth();
+    }
```

Comment:
```suggestion
    public Game(int nplayers){
        log = "";
        char number;
        // Number of rows and columns
        int nRows = 5, nCols = 5;
        // int exitRow = Dice.randomPos(nRows), exitCol = Dice.randomPos(nCols);
        int exitRow = 1, exitCol = 0;
        players = new ArrayList<Player>(0);
        for (int i = 0; i < nplayers; i++){
            players.add(new Player((char)(i+'0'), Dice.randomIntelligence(), Dice.randomStrength()));
        }
        currentPlayerIndex = Dice.whoStarts(nplayers);
        currentPlayer = players.get(currentPlayerIndex);
        monsters = new ArrayList<Monster> (0);
        labyrinth = new Labyrinth(nRows,nCols,exitRow, exitCol);
        configureLabyrinth();
        
    }
```
Creo que no hace falta pasar los arrays porque se inicializan dentro del propio juego, el de jugadores con el parámetro nplayers y el de monstruos en configureLabyrinth()